jcli/src/jcli_app/certificate/mod.rs

```diff
@@ -19,6 +19,8 @@ custom_error! {pub Error
     OutputInvalid { source: std::io::Error, path: PathBuf }
         = @{{ let _ = source; format_args!("invalid output file path '{}'", path.display()) }},
     InvalidCertificate { source: CertificateFromStrError } = "Invalid certificate",
+    ManagementThresholdInvalid { got: String, expected: String }
+        = @{{ format_args!("invalid management_threshold value, expected between 1 - {}, got {}", expected, got) }},
```

Comment:
```suggestion
        = "invalid management_threshold value, expected between at least {expected} but got {got}"
```

Reply:
AFAIK management_threshold can be in the range [1 - number of owners].
`expected` here contains the number of owners, probably should rename it to `max_expected` and maybe make 1 as `min_expected`.

Shouldn't the message be **at most** and not **at least**?
If yes it will look akward when `management_threshold = 0`